### PR TITLE
Enable GRPC logging

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Configures GRPC logs to use `logrus`.

## Related Issue(s)

Fixes #

## How to test

Check any go component with grpc connections, like `ws-manager`and check it contains similar output to 
``` 
 130 ❯ k logs -f -c ws-manager ws-manager-86dd89c5c8-snfhb|grep grpc
{"component":"grpc","level":"info","message":"WARNING: 2021/09/11 16:33:18 [core] Adjusting keepalive ping interval to minimum period of 10s","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:18Z"}
{"component":"grpc","level":"warning","message":"WARNING: 2021/09/11 16:33:18 [core] Adjusting keepalive ping interval to minimum period of 10s","serviceContext":{"service":"ws-manager","version":""},"severity":"WARNING","time":"2021-09-11T16:33:18Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] parsed scheme: \"\"","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] scheme \"\" not registered, fallback to default scheme","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] ccResolverWrapper: sending update to cc: {[{10.96.0.90:10590  \u003cnil\u003e 0 \u003cnil\u003e}] \u003cnil\u003e \u003cnil\u003e}","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] ClientConn switching balancer to \"pick_first\"","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Channel switches to new LB policy \"pick_first\"","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Subchannel Connectivity change to CONNECTING","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Subchannel picks a new address \"10.96.0.90:10590\" to connect","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Channel Connectivity change to CONNECTING","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Subchannel Connectivity change to READY","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:33 [core] Channel Connectivity change to READY","serviceContext":{"service":"ws-manager","version":""},"severity":"INFO","time":"2021-09-11T16:33:33Z"}
``` 

in case of errors
```
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:14 [core] Subchannel picks a new address \"ws-manager:8080\" to connect","serviceContext":{"service":"ws-proxy","version":""},"severity":"INFO","time":"2021-09-11T16:33:14Z"}
{"component":"grpc","level":"info","message":"WARNING: 2021/09/11 16:33:15 [core] grpc: addrConn.createTransport failed to connect to {ws-manager:8080 ws-manager:8080 \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial tcp 10.100.11.2:8080: i/o timeout\". Reconnecting...","serviceContext":{"service":"ws-proxy","version":""},"severity":"INFO","time":"2021-09-11T16:33:15Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:15 [core] Subchannel Connectivity change to TRANSIENT_FAILURE","serviceContext":{"service":"ws-proxy","version":""},"severity":"INFO","time":"2021-09-11T16:33:15Z"}
{"component":"grpc","level":"info","message":"INFO: 2021/09/11 16:33:15 [core] Channel Connectivity change to TRANSIENT_FAILURE","serviceContext":{"service":"ws-proxy","version":""},"severity":"INFO","time":"2021-09-11T16:33:15Z"}
{"component":"grpc","level":"warning","message":"WARNING: 2021/09/11 16:33:15 [core] grpc: addrConn.createTransport failed to connect to {ws-manager:8080 ws-manager:8080 \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial tcp 10.100.11.2:8080: i/o timeout\". Reconnecting...","serviceContext":{"service":"ws-proxy","version":""},"severity":"WARNING","time":"2021-09-11T16:33:15Z"}
{"
```

## Release Notes
```release-note
NONE
```

Note: the default level `info` should be enabled temporarely.

### CODE REVIEW

**Comment by roboquat** [comment]
@aledbf: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/content-service/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service/OWNERS)**
- **[components/supervisor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor/OWNERS)**
- **[components/workspacekit/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/workspacekit/OWNERS)**
- **[components/ws-daemon/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-daemon/OWNERS)**
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/content-service/cmd/run.go
Comment (csweichel) [inline components/content-service/cmd/run.go:42]: I reckon it would make sense to move this to `common-go/grpc`, no?

Comment (aledbf) [reply components/content-service/cmd/run.go]: done

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **csweichel** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)**
- **[components/content-service/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service/OWNERS)**
- **[components/supervisor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor/OWNERS)**
- **[components/workspacekit/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/workspacekit/OWNERS)**
- **[components/ws-daemon/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-daemon/OWNERS)**
- **[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)**
- **[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)**
- **[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by csweichel — APPROVED** [review]
LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 7d2d791004ff7e588a73eb461eea18ddfc1ade9d</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5664#pullrequestreview-752915688" title="Approved">csweichel</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [csweichel]
- ~~[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)~~ [csweichel]
- ~~[components/content-service/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service/OWNERS)~~ [csweichel]
- ~~[components/supervisor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor/OWNERS)~~ [csweichel]
- ~~[components/workspacekit/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/workspacekit/OWNERS)~~ [csweichel]
- ~~[components/ws-daemon/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-daemon/OWNERS)~~ [csweichel]
- ~~[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)~~ [csweichel]
- ~~[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)~~ [csweichel]
- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5664#issuecomment-918319079" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5664#issuecomment-918319079" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [csweichel]
- ~~[components/common-go/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/common-go/OWNERS)~~ [csweichel]
- ~~[components/content-service/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/content-service/OWNERS)~~ [csweichel]
- ~~[components/supervisor/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/supervisor/OWNERS)~~ [csweichel]
- ~~[components/workspacekit/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/workspacekit/OWNERS)~~ [csweichel]
- ~~[components/ws-daemon/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-daemon/OWNERS)~~ [csweichel]
- ~~[components/ws-manager/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-manager/OWNERS)~~ [csweichel]
- ~~[components/ws-proxy/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ws-proxy/OWNERS)~~ [csweichel]
- ~~[dev/gpctl/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/dev/gpctl/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
